dataprofiler/profilers/profiler_options.py

```diff
@@ -278,16 +284,16 @@ def is_numeric_stats_enabled(self):
         :rtype bool:
         """
         if self.min.is_enabled or self.max.is_enabled or self.sum.is_enabled \
-                or self.variance.is_enabled \
-                or self.histogram_and_quantiles.is_enabled:
+                or self.variance.is_enabled or self.histogram_and_quantiles.is_enabled \
```

Comment:
does this fit within 80 characters? If not keep it separated. We try to limit all our lines to < 80 chars